src/Plugins/Init.php

```diff
@@ -119,6 +119,6 @@ public function init(): void
      */
     private function isLaravelInstalled(): bool
     {
-        return InstalledVersions::isInstalled('laravel/laravel');
+        return InstalledVersions::isInstalled('laravel/laravel') || InstalledVersions::isInstalled('laravel/framework');
```

Comment:
I think an additional check is required (along with this one), as this would trigger on things such Laravel packages which shouldn't extend this testcase.

Reply:
Ah, that's a good point. I'm not sure what the check would be to _exclude_ those.

I guess the _other_ way to do this would be for the user to tell the init command that Laravel is installed e.g `vendor/bin/pest --init --laravel`. That approach would mean the `isLaravelInstalled` check stays the same and so couldn't break anything (like a package). It would be nicer to automate it though.

Reply:
I guess we could check if `artisan` exists maybe. 🤷🏻 That would handle most cases.

This is how Tinkerwell does it:
https://github.com/beyondcode/tinkerwell/blob/main/src/Drivers/LaravelTinkerwellDriver.php#L7

Reply:
That makes sense. Perhaps:

```php
private function isLaravelInstalled(): bool
{
   return file_exists($this->testSuite->rootPath.'/public/index.php') && file_exists($this->testSuite->rootPath.'/artisan');
}
```

... or get them all in there to cover all the bases ...

```php
private function isLaravelInstalled(): bool
{
    return InstalledVersions::isInstalled('laravel/framework') && file_exists($this->testSuite->rootPath.'/public/index.php') && file_exists($this->testSuite->rootPath.'/artisan');
}
```

... since it looks like the equivalent to Tinkerwell's path is `$this->testSuite->rootPath`.

Perhaps checking for `artisan` alone is enough? I can't imagine you'd have one without the other, or create a file called `artisan` if _not_ using Laravel.

Reply:
what if someone changed the name of that command file?

Reply:
@henzeb If they did, yep, the new check would also fail to detect Laravel. Just like Tinkerwell would fail to.

Doing _that_ seems less likely (to me anyway) than someone changing the `name` in composer.json.